amplipi/updater/static/index.html

```diff
@@ -84,6 +87,20 @@ <h1><span class="text-white">Ampli</span><span class="text-danger">Pi</span></h1
               </div>
               <a id="password" role="button" class="btn btn-lg btn-primary text-center mt-3" onClick="set_password();">Update Password</a>
             </div>
+            <div class="tab-pane fade text-left" id="support-tunnel" role="tabpanel" aria-labelledby="support-tunnel-tab">
+              <div id="support-tunnel-dialog">
+                This dialog allows you to request a support tunnel. Once requested, you must give the tunnel ID and preshared key to support, who can be reached via <a href="mailto:support@micro-nova.com">support@micro-nova.com</a>.
+              </div>
+              <div id="support-tunnel-button">
+                <a id="request-support-tunnel" role="button" class="btn btn-lg btn-primary text-center mt-3" onClick="requestSupportTunnel();">Request support tunnel</a>
+              </div>
+              <div id="support-tunnel-detail-container" class="d-none">
+                <pre id="support-tunnel-detail" class="alert alert-dark mt-3"></pre>
+                <div id="support-tunnel-detail-caption">
+                    Copy & paste the above lines in a support request.
+                </div>
```

Comment:
Unfortunately, the "copy to clipboard" mechanisms in Javascript are only available within secure contexts - localhost, and things over HTTPS. We are neither.